api/rocparser.h

```diff
@@ -275,6 +275,13 @@ extern rocDecStatus ROCDECAPI rocDecCreateVideoParser(RocdecVideoParser *parser_
 /************************************************************************************************/
 extern rocDecStatus ROCDECAPI rocDecParseVideoData(RocdecVideoParser parser_handle, RocdecSourceDataPacket *packet);
 
+/************************************************************************************************/
+//! \ingroup group_rocparser
+//! \fn rocDecStatus ROCDECAPI rocDecParserMarkForReuse(RocdecVideoParser parser_handle, int pic_idx)
```

Comment:
@rrawther please replace `rocDecParserMarkForReuse` int he comment with `rocDecParserMarkFrameForReuse` as the function name